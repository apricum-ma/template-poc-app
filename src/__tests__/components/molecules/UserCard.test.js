import React from 'react';
import { shallow } from 'enzyme';
import UserCard from '../../../components/molecules/UserCard';

describe('User Card', () => {
  it('should render card', () => {
    const wrapper = shallow(<UserCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
