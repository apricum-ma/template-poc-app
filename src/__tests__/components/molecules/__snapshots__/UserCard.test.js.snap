// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Card should render card 1`] = `ShallowWrapper {}`;
